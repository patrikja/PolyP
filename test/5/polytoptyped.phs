main = print p >>
       print (oM (fst p),oL (snd p))

oM = out :: Maybe Int -> FunctorOf Maybe Int (Maybe Int)
oL = out :: String -> FunctorOf [] Char String
p  = (,) (Just 1) "" :: (,) (Maybe Int) ([] Char)

