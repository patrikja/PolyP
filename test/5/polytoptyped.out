(Just 1,"")
(Right 1,Left ())
